jsapp/js/account/account.constants.ts

```diff
@@ -0,0 +1,29 @@
+export type UserFieldName =
+  | 'name'
+  | 'organization'
+  | 'organization_website'
+  | 'sector'
+  | 'gender'
+  | 'bio'
+  | 'city'
+  | 'country'
+  | 'require_auth'
+  | 'twitter'
+  | 'linkedin'
+  | 'instagram';
```

Comment:
This could get condensed to one line using `keyof` and `AccountFieldsValues`. You might then need to change UserFieldNames to a [Record type](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) instead of a generic.

Reply:
I like that `Record<>`, thanks for showing it to me :) I've updated the code to use it and moved some types around